students/gene/project/node_db.py

```diff
@@ -0,0 +1,131 @@
+
+import sqlite3
+from sqlite3 import Error
+
+
+class Node:

```

Comment:
at this point, a Node object doesn't have any logic -- so maybe it would be better to simply use a dict to store all that -- less boilerplate. Though if you forsee logic being added, then yes, a class is the right way to go.